### ISSUE TITLE
v16.7.0: p-autoComplete doesn't show selected item from it's panel in input

### ISSUE THREAD

**Issue by eXpertise7** [primefaces/primeng#14018]
### Describe the bug

`p-autoComplete` component does not show value in normal format selected item from it's panel. It shows `[object Object]`.

Video below:

https://github.com/primefaces/primeng/assets/17203747/e83bb499-a807-420b-8ba3-8a83339e2a13

### Environment

Angular v16.2.3, PrimeNG v16.7.0, Windows 10,

### Angular version

16.2.3

### PrimeNG version

16.7.0

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

16.20.2

### Browser(s)

Google Chrome, version 118.0.5993.118

### Steps to reproduce the behavior

1. Watch the video

### Expected behavior

Proper value that user selects in panel should appear in `p-autoComplete` input.

**Comment by cetincakiroglu** [comment]
Thanks for reporting the issue, the component works in demos. Could you please share some code example or stackblitz reproducer so we can identify it clearly?

**Comment by eXpertise7** [comment]
@cetincakiroglu I've figured out where the issue is. I wouldn't call it as a bug, but you guys have changed what `(onSelect)` `EventEmitter` from `p-autoComplete` sends.

In v16.6.0 and prior `p-autocomplete` emits, only the value with object:
![stackblitz_2](https://github.com/primefaces/primeng/assets/17203747/a3719f73-4d6e-4fd6-b369-6bec0fb79e7f)

In newest v16.7.0 `p-autocomplete` emits,  object with two properties (objects): originalEvent and value.
![stackblitz_1](https://github.com/primefaces/primeng/assets/17203747/e9f07963-0472-4cb0-89d3-e41273d6a0c1)

I just needed to add one more nested value property in the variable (`this.location` below) that's bound to `[(ngModel)]` of `p-autoComplete`.

So in v16.6.0, this has worked: `this.location = { name: value.name };`

in v16.7.0, this is needed: `this.location = { name: value['value'].name };`

Btw, I started creating StackBlitz example, but it's time consuming to do all of that.

You guys can close this issue if you want.
